projects/smart-table/src/lib/components/column-selector/column-selector.component.scss

```diff
@@ -0,0 +1,10 @@
+li {
+  cursor: move;
+}
+::ng-deep .cdk-drop-list-dragging .cdk-drag {
```

Comment:
mag ng-deep nog? :D

Reply:
Zeker, dat is ook de selector geworden in ng 9 & 10 als ik me niet vergis